### ISSUE TITLE
Error during compile

### ISSUE THREAD

**Issue by robertflesch** [operando/advertising_id#8]
Trying this plugin for the first time. Ran into an error I don't understand, and googling it doesn't reveal much.
```
D:\dev\purified\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java:18: error: constructor AdvertisingIdPlugin in class AdvertisingIdPlugin cannot be applied to given types;
    flutterEngine.getPlugins().add(new com.os.operando.advertisingid.AdvertisingIdPlugin());
                                   ^
  required: Registrar
  found: no arguments
  reason: actual and formal argument lists differ in length
1 error
```
coding invoking it
```
try {
      advertisingId = await AdvertisingId.id(true);
    } on PlatformException {
      advertisingId = 'Failed to get platform version.';
    }
```
thoughts?

**Comment by mattia098** [comment]
Same problem

**Comment by gustavo-perini** [comment]
Same issue here

**Comment by perez-danilo** [comment]
Same issue here

**Comment by HarshaiOS1** [comment]
same issue here.. did someone got any solution?

**Comment by joaocunhajeronimo** [comment]
I faced the same issue.
I believe `v1.1.0` of this plugin is currently not compatible with Android v2 embedding.
If your project is using Android v2 embedding ([as it should](https://flutter.dev/docs/release/breaking-changes/android-v1-embedding-create-deprecation)), you'll need to make sure all plugins you're using support it.
This plugin has code supporting it (see [respective PR](https://github.com/operando/advertising_id/pull/7)) but it was added after `v.1.1.0`.
